dynawo/sources/Models/Modelica/Dynawo/NonElectrical/Blocks/CMakeLists.txt

```diff
@@ -6,7 +6,7 @@
 # file, you can obtain one at http://mozilla.org/MPL/2.0/.
 # SPDX-License-Identifier: MPL-2.0
 #
-# This file is part of Dynawo, an hybrid C++/Modelica open source time domain simulation tool for power systems.
+# This file is part of Dynawo, an hybrid C++/Modelica open source suite of simulation tools for power systems.
```

Comment:
Maybe it was unclear but you should not modify the copyright of existing files

Reply:
Understood.